add_me_to_your_PATH/julia_pod

```diff
@@ -166,14 +168,21 @@ success "==== Pod $podname is RUNNING! ====\n\n"
     echo ""
 }
 
+if [[ -n ${PORT_FORWARD} ]]; then
+    sleep 1
+    kubectl port-forward pod/$podname $PORT_FORWARD >/tmp/kubectl-port-forward-$RUNID.log &
```

Comment:
Looks like you forgot to specify the namespace
```suggestion
    kubectl port-forward -n $KUBERNETES_NAMESPACE pod/$podname $PORT_FORWARD >/tmp/kubectl-port-forward-$RUNID.log &
```

Reply:
oh right it's not necessary for our usual setup but might be for others